File: Iniciante/1036 - Fórmula de Bhaskara/C#/1036 - Fórmula de Bhaskara.cs
using System;
using System.Globalization; // formata os valores para en-us (.)

class URI {
    static void Main() {
        string[] calculo = Console.ReadLine().Split(' ');
            double A = double.Parse(calculo[0]);
            double B = double.Parse(calculo[1]);
            double C = double.Parse(calculo[2]);

        double delta = Math.Pow(B, 2)-4*A*C;
        double x1 = (-B+Math.Sqrt(delta))/(2*A);
        double x2 = (-B-Math.Sqrt(delta))/(2*A);

        if(A == 0 || delta < 0) {
            Console.WriteLine("Impossivel calcular");
        } else {
            Console.WriteLine("R1 = " + x1.ToString("0.00000", System.Globalization.CultureInfo.InvariantCulture));

            Console.WriteLine("R2 = " + x2.ToString("0.00000", System.Globalization.CultureInfo.InvariantCulture));
        }
    }
}

File: Iniciante/1036 - Fórmula de Bhaskara/Python/1036 - Fórmula de Bhaskara.py
import math

# recebe os valores na mesma linha e atribui às variáveis
A, B, C = input().split(' ')

# converte os valores para o tipo float
A = float(A)
B = float(B)
C = float(C)

# cálculo de delta
delta = (B**2)-4*A*C

# validação + saída
if A == 0 or delta < 0:
    print('Impossivel calcular')

else:
    # cálculo de bhaskara
    x1 = (-B+math.sqrt(delta))/(2*A)
    x2 = (-B-math.sqrt(delta))/(2*A)

    # saída
    print('R1 = {:.5f}'.format(x1))
    print('R2 = {:.5f}'.format(x2))